build-fixer.mjs

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import fs from "fs";
+const args = process.argv;
+const command = args[2];
+import { exec } from "child_process";
+
+
+if (!command) {
+  console.log("No command specified");
+  process.exit(1);
+}
+
+function deleteHelper(path) {
+
+  if (fs.existsSync(path)) {
+    const lstat = fs.lstatSync(path);
+    if (lstat.isFile()) {
+      fs.unlinkSync(path);
+    } else {
+      const files = fs.readdirSync(path);
+      files.forEach(file => {
+        deleteHelper(`${path}/${file}`);
+      });
+      fs.rmdirSync(path);
+    }
+  }
+}
+
+let commandResult;
+const env = {
+  NODE_ENV: "production",
+};
```

Comment:
I think we don't need this, see my next comment

Reply:
yup